Jenkinsfile

```diff
@@ -72,10 +72,8 @@ def buildManifest = {String manifest, String bitbake_image ->
     sh "vagrant destroy -f || true"
 }
 
-node("DockerCI") {
-    buildManifest("pelux-intel.xml", "core-image-pelux")
-}
+def builders = [:]
+builders['core'] = { node("DockerCI") { buildManifest("pelux-intel.xml", "core-image-pelux") }}
+builders['qt']   = { node("DockerCI") { buildManifest("pelux-intel-qt.xml", "core-image-pelux-qt") }}
```

Comment:
I am not sure if the "qt" builder name makes it obvious that in fact it will include the core image, plus Qt, plus Qt Auto. The difference in `core-image-pelux` and `core-image-pelux-qt` is actually not Qt but Qt Auto. I would suggest the builder names to be changed to something more sensible, do you agree with this @rpannek ?